pulse-physiology_FHIR/ppe_fhir.ipynb

```diff
@@ -125,18 +173,74 @@
     "            print(\"Unable to load initial state file\")\n",
     "        else:\n",
     "            self.results = self.pulse.pull_data()\n",
+    "        self.data_management.save_csv(self.results)\n",
+    "\n",
+    "    # Advances the time by 1 second and records the patient's state after each action\n",
+    "    def advance_time_and_record(self, start, seconds, action=None):\n",
```

Comment:
if advance_time_and_record is meant to be a private function, please use the naming convention to start with an _
I was under the impression this was a public function on the SimulatorCore.

Reply:
I made an appropriate access modifier to each function with the comming commit.